src/lib/util.py

```diff
@@ -77,6 +83,14 @@ def parse_string_of_key_value_pairs(str1):
     dict1[k] = dict1.get(k, []) + [v]
   return dict1
 
+def replace_dashes_in_dict_keys_with_underscores(dict1):
+  ''' replace every dash in dict1's keys with underscore '''
+  dict2 = {}
+  for k,v in dict1.items():
+    k2 = k.replace('-', '_')
+    dict2[k2] = v
+  return dict2
```

Comment:
The more idiomatic way of doing this:

```python
return {key.replace('-', '_'): value for key, value in dict1.items()}
```

Reply:
wow, i didn't know dict comprehension existed. thanks for the tip!